packages/mdc-list/foundation.js

```diff
@@ -197,6 +194,8 @@ class MDCListFoundation extends MDCFoundation {
    * @param {number} listItemIndex
    */
   handleKeydown(evt, isRootListItem, listItemIndex) {
+    if (this.adapter_.getListItemCount() === 0) return;
```

Comment:
When does this case happen? (Is it even possible for keyboard events to happen if there are no list items to focus?)

Reply:
Removed this condition.